### ISSUE TITLE
"Station SSIDs" shows other stations with similar callsigns

### ISSUE THREAD

**Issue by arodland** [ge0rg/aprsdroid#310]
Basically, it's doing a prefix match, instead of a complete (SSID-insensitive) callsign match.

If I tap on KC2G-9, I will see KC2G and KC2G-15 in "Station SSIDs", which is expected, but I will also see KC2GOW and KC2GYU-9, which is not.

**Comment by ge0rg** [comment]
Yes, this is a long-standing issue based on the fact that I do not have a dedicated `callsign` field but only one for callsign + SSID, and I end up doing a `LIKE "$callsign%"` query to obtain all SSIDs. This is even worse for EchoLink stations because you'll get a very long list of `EL-*` :(

I really need to add code to properly normalize that data when receiving it, so that I don't need to rely on a database query hack :)

**Comment by arodland** [comment]
You could still use something like `= '$callsign' OR LIKE '$callsign-%'` I
think. Wouldn't fix the "EL-" case but it would help the ordinary one.
